README.md

```diff
@@ -35,6 +35,24 @@ This tutorial will provide a gentle introduction to all three areas. Additionall
 * Introduction to `deepcell-tf`
 * Training a model in Google Colab
 
+## Publications
+
+To learn more about the various systems and software that comprise DeepCell, please refer to the publications below. Relevant links are highlighted below each publication.
+
```

Comment:
Let's drop all the periods from bullet points. Right now we have a mix, which is bad, and generally no periods is best convention.

Reply:
👍